src/app/containers/OptimizelyPageViewTracking/index.jsx

```diff
@@ -3,15 +3,14 @@ import { OptimizelyContext } from '@optimizely/react-sdk';
 import { RequestContext } from '#contexts/RequestContext';
 import useOptimizelyScrollDepth from '#hooks/useOptimizelyScrollDepth';
 import useOptimizelyVariation from '#hooks/useOptimizelyVariation';
-
-const IMPROVED_PROMO_EXPERIMENT_ID = 'improved_promos';
+import OPTIMIZELY_CONFIG from '#lib/config/optimizely';
 
 const OptimizelyPageViewTracking = () => {
   const { isAmp } = useContext(RequestContext);
   const { optimizely } = useContext(OptimizelyContext);
   const [pageViewSent, setPageViewSent] = useState(false);
 
-  const promoVariation = useOptimizelyVariation(IMPROVED_PROMO_EXPERIMENT_ID);
+  const promoVariation = useOptimizelyVariation(OPTIMIZELY_CONFIG.featureId);
```

Comment:
General question regarding `useOptimizelyVariation`, if we pass in a single featureId, won't page view tracking work only on that experiment? What if we multiple experiment running, will page tracking work if we pass in a single feature id?

Reply:
Sorry just seen this.

Yea unfortunately thats the limitation with our current implementation. At the moment, there are no plans to run multiple experiments at once, but it is something we'll need to refactor if we do.